### DISCUSSION TITLE
MindAR + Aframe particles system component ?

### DISCUSSION THREAD

**Discussion by RajSriShanker** [hiukim/mind-ar-js#92] [Q&A]
Hi, love using MindAR, however I'm having trouble incorporating aframe particle system component within my code. Has anyone used particles with MindAR before? If so do you have a basic example? Thanks!

https://www.npmjs.com/package/aframe-particle-system-component

**Comment by vincentfretin** [comment]
I replied in https://github.com/hiukim/mind-ar-js/issues/93#issuecomment-998126352
